src/renderer/features/discord-rpc/use-discord-rpc.ts

```diff
@@ -77,6 +77,23 @@ export const useDiscordRpc = () => {
                     type: discordSettings.showAsListening ? 2 : 0,
                 };
 
+                if (discordSettings.linkLastfm && song?.artistName) {
+                    activity.stateUrl = 'https://www.last.fm/music/' + song.artists[0].name;
+                    activity.detailsUrl =
+                        'https://www.last.fm/music/' +
+                        song.albumArtists[0].name +
+                        '/' +
+                        (song.album || '_') +
+                        '/' +
+                        song.name;
+                }
+
+                if (discordSettings.linkMusicbrainz && song?.mbzId) {
+                    if (song.serverType == 'subsonic')
+                        activity.detailsUrl = 'https://musicbrainz.org/recording/' + song.mbzId;
+                    else activity.detailsUrl = 'https://musicbrainz.org/track/' + song.mbzId;
```

Comment:
Please wrap these with `{}` (or do it via a tuple, `song.serverType == 'subsonic' ? ...`)